cypress/integration/my-orcid/sidebar-add-websites-negative-testing.e2e.js

```diff
@@ -0,0 +1,506 @@
+
+    //clean up state
+    cy.cleanWebsites()
+
+    //reload page to reflect clean up
+    cy.reload()
+  })
+
+  it('URLs in mixedcase are valid', function () {
+    //click on edit pencil for websites section
+    cy.get('#websites-panel').within(($myPanel) => {
+      cy.get('#edit-button').click()
+    })
+    cy.get('#add-link').click()
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.mixedCaseURL)
+    cy.get('#save-websites-button').click()
+
+    //verify URL case was not modified by the system in any way
+    cy.get('#websites-panel')
+      .within(($section) => {
+        cy.get('[class="line"]')
+      })
+      .should('contain', testingData.sidebarWebsitesURL.mixedCaseURL)
+
+    //clean up state
+    cy.cleanWebsites()
+
+    //reload page to reflect clean up
+    cy.reload()
+  })
+
+  it('Internationalized (valid) URLs are not allowed', function () {
+    /* Business rule: valid internationalized URLs are NOT supported */
+    //click on edit pencil for websites section
+    cy.get('#websites-panel').within(($myPanel) => {
+      cy.get('#edit-button').click()
+    })
+    cy.get('#add-link').click()
+    cy.get('#description-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.titleURL)
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL1)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL2)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL3)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#url-input')
+      .clear()
+      .type(testingData.sidebarWebsitesURL.internationalURL4)
+    cy.get('#save-websites-button').click()
+    //verify error message is displayed
+    cy.get('mat-error').should(
+      'have.text',
+      testingData.errorMessages.invalidURL
+    )
+
+    cy.get('#cancel-websites-button').click()
+  })
+
+  it('Error messages display according to the language selected', function () {
+    //switch to Spanish
+    cy.get('[aria-label="language"]').click()
+    cy.get('[id^=cdk-overlay]').within(($menuLanguage) => {
+      cy.get('button[role="menuitem"]')
+        .contains(testingData.errorMessages.displayLanguageSpanish)
+        .click()
+    })
+    cy.wait(3000) //wait for language to refresh
```

Comment:
We can avoid using the wait 
```suggestion
   cy.get('[lang="es"]') //wait for language to refresh
```

Reply:
Done